js/schematic.js

```diff
@@ -211,7 +210,7 @@
         });
 
       signal.on("query", function(question) {
-        question.colorScale = colorScale[question.section][question.legend];
+        question.colorScale = colorScale2[question.section][question.legend];
```

Comment:
Do we need to keep both `colorScale` and `colorScale2` in the picture, or can we get rid of the old `colorScale` and rename `colorScale2` to `colorScale`? This would be a good change to see before merging.

I believe the following line in the old `colorScale` definition is causing this `d3.scaleSingular is not a function` error that I'm seeing:

```js
colorScale["{{ section[0] }}"]["{{ legend.name }}"] = d3.scale{{ legend.type | capitalize }}()
```